make-dist

```diff
@@ -202,12 +210,14 @@ pmdk_version=1.10
 download_pmdk $pmdk_version 08dafcf94db5ac13fac9139c92225d9aa5f3724ea74beee4e6ca19a01a2eb20c \
                https://github.com/pmem/pmdk/releases/download/$pmdk_version
 build_dashboard_frontend
+build_mgr_pip_pkgs
 generate_rook_ceph_client
 for tarball in $outfile.version   \
                $outfile.boost     \
                $outfile.liburing  \
                $outfile.pmdk  \
                dashboard_frontend \
+               mgr_pip_pkgs \
```

Comment:
i'm not really familiar with this `make-dist` script, but all the stuff about tarballs makes it look like it's only for the source rpm ([ceph-18.0.0-2775.ge6614bf9.el9.src.rpm](https://1.chacra.ceph.com/r/ceph/wip-pereman2-testing-2022-09-05-1259/e6614bf94d54bafb698cdcb6066c7e24e341eb9a/centos/9/flavors/default/SRPMS/ceph-18.0.0-2775.ge6614bf9.el9.src.rpm)) and not the normal rpm packages like [ceph-mgr-18.0.0-2775.ge6614bf9.el9.x86_64.rpm](https://1.chacra.ceph.com/r/ceph/wip-pereman2-testing-2022-09-05-1259/e6614bf94d54bafb698cdcb6066c7e24e341eb9a/centos/9/flavors/default/x86_64/ceph-mgr-18.0.0-2775.ge6614bf9.el9.x86_64.rpm) that get installed by teuthology and upstream users

Reply:
this ceph-mgr rpm only contains:
```
$ rpm -ql ceph-mgr-18.0.0-2775.ge6614bf9.el9.x86_64.rpm 
/usr/bin/ceph-mgr
/usr/lib/.build-id
/usr/lib/.build-id/6f
/usr/lib/.build-id/6f/6a67056bbcf220ae9dc3621bfaab2514f5f5e8
/usr/lib/systemd/system/ceph-mgr.target
/usr/lib/systemd/system/ceph-mgr@.service
/usr/share/ceph/mgr
/usr/share/ceph/mgr/mgr_module.py
/usr/share/ceph/mgr/mgr_util.py
/usr/share/ceph/mgr/object_format.py
/var/lib/ceph/mgr
```

Reply:
I thought it was build with the same source tarball, how are the source files provided to other rpms?

Reply:
for compiled languages like c/c++, rpms generally don't need to contain source files - only the compiled binaries

source rpms are for users that want to compile ceph from source. downstream distros can also use these source tarballs to package ceph, so they can add additional patches before compiling

Reply:
okay great; so which ceph package(s) should all of those `__pypackages__` files end up in? if `ceph-mgr-modules-core` is the right place, i assume we'll need to add the files to `ceph.spec.in`:
```diff
 %files mgr-modules-core
 %dir %{_datadir}/ceph/mgr
+%{_datadir}/ceph/mgr/__pypackages__
```
and `debian/ceph-mgr-modules-core.install`
```diff
+usr/share/ceph/mgr/__pypackages__
 usr/share/ceph/mgr/alerts
```
i'm not sure whether or not that works with entire directories, maybe it needs a `/*` at the end